### PR TITLE
Adding profile properties data type to Snowflake doc

### PR DESCRIPTION
**Pull request by chrisseldo**
# Amplitude Developer Docs PR
Adding profile properties to Snowflake data type support docs

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?
ASAP :)

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [X] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [X] My documentation follows the style guidelines of this project.
- [X] I previewed my documentation on a local server using `mkdocs serve`.
- [X] Running `mkdocs serve` didn't generate any failures.
- [X] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Comment by chrisseldo** [comment]
@bgiori - would you mind just merging this please?

File: docs/data/sources/snowflake.md [hunk 1]
```diff
@@ -104,6 +104,15 @@ You must include the mandatory fields for the data type when creating the SQL qu
 | `update_time_column` | No (Yes if using time based import) | TIMESTAMP_NTZ | 2013-04-05 01:02:03.000 |
 
 Each group property in `group_properties` would be applied to every group in `groups`
+### Profile properties
```
Comment (justin-fiedler) [inline docs/data/sources/snowflake.md:107]: nit: please fix MegaLinter errors

```suggestion

### Profile properties
```

**Review by justin-fiedler — APPROVED** [review]

**Comment by kevinpagtakhan** [comment]
@chrisseldo do we still need this PR? if so, looks like we can merge already?

File: docs/data/sources/snowflake.md [hunk 2]
```diff
@@ -105,6 +105,16 @@ You must include the mandatory fields for the data type when creating the SQL qu
...
 
+### Profile properties
+
+| Column name (must be lowercase) | Mandatory | Column data type | Example |
+|---|---|---|---|
+| `user_id` | Yes | VARCHAR | "user123" |
+| `property_name_1` | Yes | (key value) VARCHAR: VARCHAR | "Title": "Data Engineer" |
+| `property_name_1` | Yes | (key value) VARCHAR: VARCHAR | "City": "San Francisco" |
+
+Note: Profile properties are only supported for known Amplitude users. Therefore, each profile property must be accompanied by a user identifier (user_id).
```
Comment (markzegarelli) [inline docs/data/sources/snowflake.md:116]:
```suggestion
Amplitude supports profile properties for known users. A `user_id` value must accompany each profile property.
```

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**